tests/feature/test_wrong.py

```diff
@@ -50,4 +50,4 @@ def test_wrong():
     )
     result = pytester.runpytest()
     result.assert_outcomes(errors=1)
-    result.stdout.fnmatch_lines("*FeatureError: Multiple features are not allowed in a single feature file.*")
+    result.stdout.fnmatch_lines("*FeatureError: *")
```

Comment:
what kind of errors would we show to the user in this case? I think it's important to make sure we return a decent error message

Reply:
Ah, the error messages are not nice (from a human readability perspective).

I forgot to go back and now just throw the raw error message.  I am just adding an error message parser to map the parser errors to throw the same errors are before, and a few more that weren't previously covered.